Loops/README.md

```diff
@@ -0,0 +1,143 @@
+# Loops in Golang
```

Comment:
Thanks, @Harsh4902 for reviewing, and thank @bvvnmanikanta raising PR, as you are there can you make one more change, center align this Title you can take reference from other readme

Reply:
@Yashsharma1911, center aligned the Title.